### DISCUSSION TITLE
Is repository still alive?

### DISCUSSION THREAD

**Discussion by davidgv** [scivision/PyLivestream#87] [Q&A] [answered]
I see that repository stop activity (commits) a long time ago. Is it still maintained, or is it just that everything works and you don't need to touch anything?

**Comment by scivision** [comment, marked as answer]
I usually update this project a couple times a year, but haven't taken the time lately. I did just now replace the awkward scripts with a `python -m pylivestream.camera` etc for use from command line, which is more robust.

But yes the concept is pretty stable, except as web streaming providers close or make major changes. 
It was just something I intended to be a personal use thing that people really liked.

**Reply by davidgv** [reply]
Thanks for your answer and your work.
